README.md

```diff
@@ -57,23 +58,22 @@ bin/cake server
 
 Note that this should never be used for a production site!
 
-### Experimental Docker Alternative
+### Docker Quick Start (Experimental!)
```

Comment:
Should this be moved under `Development`? It's awkward because the "Advanced" subheading suggests it's _not_ for development.

Reply:
I would agree.  It's a little clunky in the middle there and the readme would probably flow best if we stuck the entire docker chunk near the bottom by itself.